ocp_utilities/infra.py

```diff
@@ -354,3 +357,32 @@ def create_icsp(icsp_name, repository_digest_mirrors):
     )
     icsp.deploy()
     return icsp
+
+
+def update_pull_secret(new_content_dict, secret_name, secret_namespace):
```

Comment:
the name is update_ ... but you will create it if not exists, this needs to be rename

Reply:
`create_update_pull_secret` maybe @myakove ?